### DISCUSSION TITLE
Curation report Mar 13 - Mar 17

### DISCUSSION THREAD

**Discussion by jmillanacosta** [wikipathways/GPML-Repo#119] [Curation reports]
Hi everybody, I am done with my first curation week.

* Added disease ontology terms to some pathways 
* Added  references to WP5329 from the literature and sources cited in the description. Since it did not specify which sources were used for specific parts, I referenced all original sources used also in the Reactome Cholesterol Synthesis pathway.
* Fixed text encoding for references (only bad in classic WikiPathways), but maybe this isn't needed? @egonw 
* Changed nodes used to compose the legend into graphical elements (WP5343)
* Question: how to annotate a protein family (here, GeneProduct "RPS6K") if there's no term for it? Or is it left without crossreference in this case?) https://classic.wikipathways.org/index.php/Pathway:WP722
![image](https://user-images.githubusercontent.com/83466805/225969491-f6bdef3d-b0ef-4701-b153-0d1aeb15dce5.png)

**Comment by AlexanderPico** [comment]
Thanks, @jmillanacosta! Great fixes and questions.

I looked into RPS6K and don't have an immediate suggestion or fix. We tend to leave this up to authors, but it's not ideal to have a datanode annotated as GeneProduct, but not have an identifier. I think that will register as a curation bug, in fact. So, maybe make it a graphical element? Or just leave it.  I don't have a strong opinion here :)

**Reply by jmillanacosta** [reply]
Right, thanks!

**Comment by AlexanderPico** [comment]
@jmillanacosta By the way, you now have a profile page on the new site. Feel free to click the edit icon to add more details: https://www.wikipathways.org/authors/Jmillanacosta.html

**Comment by khanspers** [comment]
I created an issue on wikipathways-help for this, so that we can include it in discussions during the next Community Curation meeting: https://github.com/wikipathways/wikipathways-help/issues/26

**Reply by jmillanacosta** [reply]
Thank you!
